build_tools/github/test_windows_wheels.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+set -x
+
+PYTHON_VERSION=$1
+BITNESS=$2
+
+if [[ "$PYTHON_VERSION" == "36" || "$BITNESS" == "32" ]]; then
+    # For Python 3.6 and 32-bit architecture use the regular
+    # test command (outside of the minimal Docker container)
+    pytest --pyargs sklearn
+    python -m threadpoolctl -i sklearn
+else
+    docker run -e SKLEARN_SKIP_NETWORK_TESTS=1 \
```

Comment:
I think you should also propagate the `OMP_NUM_THREADS` and `OPENBLAS_NUM_THREADS` variables.